Gemfile

```diff
@@ -0,0 +1,22 @@
+source 'https://rubygems.org'
+
+ruby '2.4.0'
```

Comment:
lock file won't protect you from unexpected gem update if you type `bundle update sinatra`